test/domain_test.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'test_helper'
+require 'mocha/mini_test'
```

Comment:
@davesims you missed moving these requires to `test_helper` to consolidate them.